src/Tasks/Microsoft.NET.Build.Tasks/ConflictResolution/ILog.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    public enum LogImportance
+    {
+        Low = MessageImportance.Low,
+        Normal = MessageImportance.Normal,
+        High = MessageImportance.High
+    }
+
+
+    public interface ILog
```

Comment:
Feel free to replace this with whatever logging abstraction is appropriate for your task DLL.